app/Http/Controllers/ApplicantProfileController.php

```diff
@@ -25,7 +25,7 @@ class ApplicantProfileController extends Controller
      * @param  \App\Models\Applicant $applicant
      * @return \Illuminate\Http\Response
      */
-    public function show(Request $request, Applicant $applicant)
+    public function show(Request $request, Applicant $applicant): \Illuminate\Http\Response
```

Comment:
Be careful typehinting responses on controller actions, especially when returning a view. The Laravel view helper can return one of two different objects, and neither are of type `\Illuminate\Http\Response`.
If you put a mixed/optional typehint in the docblock, phpcs won't warn you about typehinting the function itself.
The danger comes from PHP just exploding if the typed return doesn't match what is actually returned at runtime.

Reply:
You're right that Response is the wrong type. I think its safe to use `\Illuminate\View\View` instead, as the view helper only returns the factory when you pass 0 arguments.